carp.data.core/src/jvmTest/kotlin/dk/cachet/carp/data/infrastructure/db/SQLiteStudyDataRepositoryTest.kt

```diff
@@ -0,0 +1,73 @@
+package dk.cachet.carp.data.infrastructure.db
+
+import dk.cachet.carp.common.application.UUID
+import dk.cachet.carp.data.application.CollectedDataQuery
+import kotlinx.coroutines.test.runTest
+import kotlinx.datetime.Instant
+import kotlin.test.*
+
+class SQLiteStudyDataRepositoryTest
```

Comment:
Look at how I use interfaces to test against for my in memory implementations. The same tests should be able to run for different concrete instances. Core's primary task is clearly defining the contract, not providing the implementation.